libkbfs/journal_md_ops.go

```diff
@@ -42,25 +42,12 @@ func (j journalMDOps) getHeadFromJournal(
 	ctx context.Context, id TlfID, bid BranchID, mStatus MergeStatus,
 	handle *TlfHandle) (
 	ImmutableRootMetadata, error) {
-	bundle, ok := j.jServer.getBundle(id)
+	tlfJournal, ok := j.jServer.getTLFJournal(id)
 	if !ok {
 		return ImmutableRootMetadata{}, nil
```

Comment:
No -- journalMDOps overlays the MDs from the journal on top of the 'regular' MDOps (talking to the MDServer), so if there's no journal at all for a particular TLF, then that just means that it falls back to the server.